apps/gp2-server/test/validation/working-group.validation.test.ts

```diff
@@ -0,0 +1,107 @@
+import { gp2 } from '@asap-hub/model';
+import { validateWorkingGroupPatchRequest } from '../../src/validation/working-group.validation';
+
+describe('Working Group validation', () => {
```

Comment:
this is a little against the current pattern by which we test validation on a route level. That way we are not only testing that the validation rules are fine but also that it is attached correctly to the router. We often extract and manipulate data that comes out of the validation so I think that's much better to do it at that level.

I see little value in those tests on their own like this

Reply:
the tests were really helpful when trying to understand if we were covering all the logic. 

I'll remove them